src/models/SimpleEvent.ts

```diff
@@ -0,0 +1,12 @@
+import { Country } from './Country'
+
+export interface SimpleEvent {
+  id: number
+  name: string
+  dateStart?: number
+  dateEnd?: number
+  prizePool: string
+  teams?: number
+  location: Country
+  host?: string
```

Comment:
Please make this property an enum, since there is a limited number of possible values - Major, International LAN, Regional LAN, Online, Local LAN and Other. Also host is not a very good name for that. `type` is probably more accurate, even the filter in HLTV is `?eventType=ONLINE` for example.

Reply:
Added the enum